cmd/thanos/rule_test.go

```diff
@@ -10,6 +10,46 @@ import (
 	"github.com/pkg/errors"
 )
 
+func Test_parseFlagLabels(t *testing.T) {
+	var tData = []struct {
+		s   []string
+		err error
+	}{
+		{
+			s:   []string{`labelName="LabelVal"`, `_label_Name="LabelVal"`, `label_name="LabelVal"`, `LAb_el_Name="LabelValue"`, `lab3l_Nam3="LabelValue"`}, // Valid
+			err: nil,
```

Comment:
nil is default value for `error` so we can remove this. Also `Valid` comment is not needed (: lack of error shows that correctly.